source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -336,14 +368,13 @@ let parse source_jsons =
               code;
               message_format;
             }
-            ::
-            {
-              Rule.sources = second_sources;
-              sinks = List.map second_sinks ~f:(fun sink -> Sinks.TriggeredPartialSink sink);
-              name;
-              code;
-              message_format;
-            }
+            :: {
+                 Rule.sources = second_sources;
+                 sinks = List.map second_sinks ~f:(fun sink -> Sinks.TriggeredPartialSink sink);
+                 name;
+                 code;
+                 message_format;
+               }
```

Comment:
Hmm our auto-formatter disagrees with this. Are we using the same one?

Reply:
Yes!